src/main/kotlin/lotto/domain/Numbers.kt

```diff
@@ -0,0 +1,12 @@
+package lotto.domain
+
+class Numbers(
+    val issueNumbers: List<Int> = Ticket.LOTTO_RANGE.shuffled().take(Ticket.LOTTO_COUNT).sorted(),
+    val bonusNumber: Int = 0
+) {
+    fun size() = this.issueNumbers.size
+
+    fun duplicateSize() = this.issueNumbers.toSet().size
+
+    fun getMatchingNumbers(numbers: Numbers) = this.issueNumbers.filter { numbers.issueNumbers.contains(it) }.size
```

Comment:
```suggestion
    fun getMatchingNumbers(numbers: Numbers) = this.issueNumbers.count { numbers.issueNumbers.contains(it) }
```

`count`를 활용할 수 있습니다 :)